CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ in the detailed section referring to by linking pull requests or issues.
 * Fix race condition in `ContractNegotiationIntegrationTest` (#1505)
 * Fix for change in Cosmos DB behavior on missing sort fields (#1514)
 * Effectively removed default LIMIT in SQL Contract Def Store (#1515)
+* Sending data to an AWS S3 presigned upload url does not work (#1643)
```

Comment:
Minor remark: A changelog entry and PR title is about documenting what was changed/added/removed to the implementation, not repeating the bug statement from the issue. Pls take a look at the other entries/PRs.

Reply:
Thank you for the hint @juliapampus . Is it better like "Fixed failure in uploading files to a presigned AWS S3 upload url (#1643)" ?